app/views/teaching_events/about_ttt_events.html.erb

```diff
@@ -0,0 +1,55 @@
+<% @page_title = "About train to teach events" %>
+<%= render Header::BreadcrumbComponent.new %>
+
+<div class="about-ttt-events">
+  <header>
+    <h1>
+      <span>What is a</span><br/>
+      <span>Train to Teach event?</span>
+    </h1>
+
+    <%= image_pack_tag "media/images/content/event-signup/birmingham-event-3.jpg" %>
+  </header>
+
+
+  <article>
+    <p>
+      Train to Teach events are designed to give you an insight into what it’s
+      like to be a teacher, as well as the practical information you need to get
+      onto a teacher training course.
+    </p>
+
+    <p>
+      Events are run by the Department for Education and are impartial. We do
+      invite local training providers along to give you an idea of what they
+      offer, but our goal is to answer your questions and give you the confidence
+      to take your next step.
+    </p>
+
+    <p>
+      To attend, you need to register. Spaces are limited, so the sooner you sign
+      up, the better.
+    </p>
+
+    <p>
+      <a href="/teaching-events" class="button">Register for a Train to Teach event</a>
```

Comment:
Can this button go to a filtered list of events ie with TTT events only selected (this is what the user would expect?)